profiler/src/CMakeLists.txt

```diff
@@ -17,6 +17,10 @@ if(GZ_PROFILER_REMOTERY)
     ./Remotery/lib/Remotery.h
   )
 
+  # A few warnings to suppress, rather than modifying vendored code
+  set_source_files_properties(./Remotery/lib/Remotery.c PROPERTIES
+    COMPILE_FLAGS "-Wno-strict-prototypes -Wno-unused-but-set-variable")
```

Comment:
I think this maybe affecting windows build? This output looks relevant:

```
cl : command line error D8021: invalid numeric argument '/Wno-strict-prototypes' [C:\J\workspace\gz_common-pr-win\ws\build\gz-common6\profiler\src\gz-common6-profiler.vcxproj]
```

Reply:
Always Windows...

Reply:
Fixed by only suppressing the warnings with GCC/Clang